lib/config.js

```diff
@@ -63,7 +63,7 @@ const _config = {};
  *  - '--config-file' - A JSON config file with options inside. See above.
  * @memberof config
  */
-function parseOptions(options = {}) {
+function parse_options(options = {}) {
```

Comment:
My personal preference is to format JS default parameters like Python default parameters, e.g. `parse_options(options={})` No big deal either way.

Reply:
This is the `space-infix-ops` Eslint rule. Gives: `Infix operators must be spaced`. I will change it.

Reply:
Yeah, the `space-infix-ops` rule doesn't have a more granular configuration. Is either on or off. I will turn it back on reverting the last commit.